DUnitX.Assert.pas

```diff
@@ -359,6 +361,23 @@ class procedure Assert.AreEqual(const expected, actual: boolean; const message:
     FailFmt(SUnexpectedErrorStr ,[BoolToStr(expected, true), BoolToStr(actual, true), message], ReturnAddress);
 end;
 
+class procedure Assert.AreEqual(const expected, actual: string; const compformat: TDUnitXComparableFormatClass; const ignoreCase: boolean; const message: string);
```

Comment:
Thanks for the detailed explanation. Happy with this, I don't have any better options for adding data definition to the assert tests.